tools/include/eosio/abigen.hpp

```diff
@@ -453,14 +453,14 @@ namespace eosio { namespace cdt {
                ojson oj;
                oj["name"] = idx.name;
                oj["type"] = idx.type;
-               o["primary_key"] = oj;
+               o["primary_index"] = oj;
             } else {
                ojson o;
                o["type"] = idx.type;
                indices.insert_or_assign(idx.name, o);
             }
          }
-         o["indices"] = indices;
+         o["secondary_indices"] = indices;
```

Comment:
Why were "primary_key" and "indices" incorrect?

Reply:
The spec definition named them as `primary_index` and `secondary_indices`, I just had the wrong thing here.

Reply:
Which spec do you refer to? Thanks.